server/zanata-frontend/src/frontend/app/editor/components/SettingsOptions/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import SettingOption from '../SettingOption'
+
+const SettingsOptions = ({states, updateSettingOption}) => {
+  const settings =
```

Comment:
Unsure how to set default props for the nested props under settings and what their default values would be. I know what I have included here is incorrect, just was using it to test how storybook looks.